UCSD-E4E/Pyrenote
# Fixed clicking on spectrogram and not playing

Test:
- Click on spectrogram
- Observe playback pauses correctly and spectrogram jumps to click
- playback should start at that point
- Also double check region creation
